src/libraries/System.Formats.Tar/tests/TarReader/TarReader.TarEntry.ExtractToFile.Tests.Unix.cs

```diff
@@ -9,8 +9,10 @@ namespace System.Formats.Tar.Tests
 {
     public partial class TarReader_TarEntry_ExtractToFile_Tests : TarTestsBase
     {
-        [PlatformSpecific(TestPlatforms.AnyUnix & ~TestPlatforms.tvOS)] // https://github.com/dotnet/runtime/issues/68360
-        [ConditionalFact(nameof(IsUnixButNotSuperUser), nameof(IsNotLinuxBionic))]
+        [PlatformSpecific(TestPlatforms.AnyUnix)]
```

Comment:
In the above .Unix.cs file there's no PlatformSpecific on the test... I assumed that was because that .Unix.cs file is only compiled into unix builds, but then here this is also a .Unix.cs and this one does have PlatformSpecific...?